specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/computeRPCommon.json

```diff
@@ -1053,6 +1053,18 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add a data disk to a virtual machine. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "description": "Specifies the DiskController Type",
```

Comment:
Specifies the disk controller type configured for the VM. This property is only supported for Virtual Machines whose operating system disk and and VM size supports hyperVGeneration: V2. <br> For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.
<br><br> Minimum api-version: 2020-08-01